inc/performance_optimizations/namespace.php

```diff
@@ -17,6 +15,9 @@ function bootstrap() {
  *
  */
 function increase_set_time_limit_on_async_upload() {
+	if ( ! isset( $_SERVER['REQUEST_URI'] ) || strpos( $_SERVER['REQUEST_URI'], '/wp-admin/async-upload.php' ) === false ) {
+		return;
+	}
 	if ( ini_get( 'max_execution_time' ) < 120 ) {
```

Comment:
I'd flip this as well; I'd also separate these out into multiple if statements

Reply:
ah I see, so a little family of `if`s

Reply:
👨‍👩‍👧‍👦